lib/upipe-srt/upipe_srt_receiver.c

```diff
@@ -575,7 +576,7 @@ static void upipe_srt_receiver_timer(struct upump *upump)
         if (unlikely(!ubase_check(uref_clock_get_cr_sys(uref, &cr_sys))))
             upipe_warn_va(upipe, "Couldn't read cr_sys in %s()", __func__);
 
-        if (now - cr_sys <= upipe_srt_receiver->latency)
+        if (now - cr_sys <= upipe_srt_receiver->latency - rtt)
```

Comment:
Need to handle the case where rtt > latency otherwise right hand side will go to UINT64_MAX and packets will buffer infinitely.